website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -0,0 +1,40 @@
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'typescript-eslint-parser/package.json';
+
+const ID = 'typescript-eslint-parser';
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: pkg.homepage,
+  locationProps: new Set(['loc', 'start', 'end', 'range']),
+
+  loadParser(callback) {
+    require(['typescript-eslint-parser'], callback);
+  },
+
+  parse(parser, code) {
+    const opts = {
+      sourceType: 'module',
+    };
+
+    const ast = parser.parse(code, opts);
+    delete ast.tokens;
```

Comment:
Does `typescript-eslint-parser` return `.tokens`? As far as I see, it does only when you pass explicit option to do so, so there's no point in deleting this property, you can just return result as-is.